src/components/concept/_ci-ignoring-files.mdx

```diff
@@ -1,8 +1,8 @@
 By default `semgrep ci` skips files and directories such as `tests/`, `node_modules/`, and `vendor/`. It uses the default `.semgrepignore` file which you can find in the [Semgrep GitHub repository](https://github.com/returntocorp/semgrep/blob/develop/.semgrepignore). This default is used when no explicit `.semgrepignore` file is found in the root of your repository.
 
-You can copy and commit the default `.semgrepignore` file to the **root of your repository** and extend it with your own entries or write one from scratch. If Semgrep detects a `.semgrepignore` file within your repository, it does not append entries from the default `.semgrepignore` file.
+Copy and commit the default `.semgrepignore` file to the **root of your repository** and extend it with your own entries or write your `.semgrepignore` file from scratch. If Semgrep detects a `.semgrepignore` file within your repository, it does not append entries from the default `.semgrepignore` file.
```

Comment:
I wonder why we have to say "copy and commit" - this is not something the user has to do. .semgrepignore will function even when the user doesn't do it.

Reply:
(What I mean to say is, it's optional.)

Reply:
@s-santillan I agree. Feel free to change this or suggest and edit... so should this stay as:
```suggestion
Optional: Copy and commit the default `.semgrepignore` file to the **root of your repository** and extend it with your own entries or write your `.semgrepignore` file from scratch. If Semgrep detects a `.semgrepignore` file within your repository, it does not append entries from the default `.semgrepignore` file.
```

Reply:
To express this in a more TW way...